### ISSUE TITLE
Bug - featured image alt text - not editable on Spanish site

### ISSUE THREAD

**Issue by SophieAmnesty** [amnestywebsite/humanity-theme#302]
**Describe the bug**  
Translators have been unable to add the Spanish alt text to featured images on news posts.

**To Reproduce**  
1. Create a post in English
2. Create linked translation in Spanish.
3. Go to featured image and try to add alt text 
4. Can't add text

**Expected behaviour**  
Translators should be able to add the featured image alt text in the post settings. 

**Actual behaviour**  
The field seems to be uneditable.

**Link to a page where the bug is reproduced in isolation (optional)**  
https://isaidotorgstg.wpengine.com/es/wp-admin/post.php?post=104679&action=edit 

examples where the translators have had this issue:
- https://www.amnesty.org/es/latest/news/2024/07/gambia-continued-ban-on-fgm-is-good-news-but-authorities-must-urgently-address-its-root-causes/
- https://www.amnesty.org/es/latest/press-release/2024/07/finland-emergency-law-on-migration-inconsistent-with-eu-law-and-must-be-rigorously-scrutinized/ 

**Screenshots**  
![image](https://github.com/user-attachments/assets/ee06b64f-7370-4f4c-8458-71fa93242784)
 
**Platform:**  
https://www.whatsmybrowser.org/b/X0NLV

**Comment by AmnestyAM** [comment]
LGTM 
![image](https://github.com/user-attachments/assets/652882b2-bcb6-4d8e-bb8d-424e3924c144)
